spec/data-spec.js

```diff
@@ -185,6 +189,13 @@ describe("Data", () => {
 				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
 				expect(+circle.attr("cy")).to.be.closeTo(expectedCy[556][i], 1);
 			});
+
+			d3SelectAll('.bb-circles-778-889 .bb-circle').each(function (d, i) {
+				const circle = d3Select(this);
+
+				expect(+circle.attr('cx')).toBeCloseTo(expectedCx[i], 0);
+				expect(+circle.attr('cy')).toBeCloseTo(expectedCy["778.889"][i], 0);
+			});
```

Comment:
Test failed, because of the unknown API.
I miss to notice in the previous review, billboard.js changed the assertion library to Chai.
`toBeCloseTo` should be `to.be.closeTo()`
- http://chaijs.com/api/bdd/#method_closeto